src/mlpack/methods/ann/layer/batch_norm_impl.hpp

```diff
@@ -25,7 +25,8 @@ BatchNorm<InputDataType, OutputDataType>::BatchNorm() :
     eps(1e-8),
     loading(false),
     deterministic(false),
-    count(0)
+    count(0),
+    size(0)
```

Comment:
This should be introduced in the same order as the other constructor to prevent compile time warnings.